world-bank/src/Code.js

```diff
@@ -0,0 +1,111 @@
+var cc = DataStudioApp.createCommunityConnector();
+
+function getAuthType() {
+  return cc
+    .newAuthTypeResponse()
+    .setAuthType(cc.AuthType.NONE)
+    .build();
+}
+
+function getConfig(request) {
+  return {
+    configParams: [],
+  };
+}
+
+function getFields() {
+  var fields = cc.getFields();
+  var types = cc.FieldType;
+  var aggregations = cc.AggregationType;
+
+  fields
```

Comment:
Adding the Country Code field would make the connector more useful for users if we publish it. They will be able to blend the data with other data sources.